cypress/e2e/smart-table/testSmartTable.cy.js

```diff
@@ -0,0 +1,104 @@
+import SmartTablePage from '../../pageObjects/tables/smart-table/SmartTablePage';
+
+
+describe('Smart table tests', () => {
+    
+    const smartTablePage = new SmartTablePage();
+
+    beforeEach(()=>{
+        smartTablePage.navigate();
+    })
+    
+    it('verify adding a user', () => {
+        
+        const userData = {
+            id: '230823',
+            firstName: 'Serhii',
+            lastName: 'Matkobozhyk',
+            userName: '@sitro',
+            email: 'sitro@gmail.com',
+            age: '29'
+        };
+
+        smartTablePage.clickPlusButton();
+
+        smartTablePage.fillAddUserSection(userData);
+
+        smartTablePage.clickConfirmButton();
+
+        // Check ID
+        cy.get('table tbody tr:nth-of-type(1) td:nth-of-type(2)').invoke('text').then(userId => {
```

Comment:
you can chain `should` instead of using `then`
https://docs.cypress.io/api/commands/invoke#Usage